types/three/src/math/Quaternion.d.ts

```diff
@@ -138,14 +138,15 @@ export class Quaternion {
      * @return The created or provided array.
      */
     toArray(array?: number[], offset?: number): QuaternionTuple;
+    toArray(array?: QuaternionTuple, offset?: 0): QuaternionTuple;
```

Comment:
Should the return type of line 140 be updated to `number[]`?

Reply:
Yes, fixed!